### DISCUSSION TITLE
230705 시나리오, 기능 명세서 정리 회의(11:35 ~ 11:50)

### DISCUSSION THREAD

**Discussion by seokhwan-an** [woowacourse-teams/2023-shook#22] [회의]
### 회의 안건
제출할 시나리오와 기능 명세서를 제출 형태를 만든다.

**Comment by somsom13** [comment]
### 시나리오
- 최상위 페르소나
- 페르소나 관점의 시나리오

[SHOOK-사용자 시나리오.pdf](https://github.com/woowacourse-teams/2023-shook/files/11953369/SHOOK-.pdf)

### 기능 명세서
<img width="3181" alt="SHOOK-기능명세-맵핑" src="https://github.com/woowacourse-teams/2023-shook/assets/70891072/6c0ccadf-c313-438a-9d96-bb76c275c4ad">
